__tests__/integrations/GoogleAds/GoogleAds.test.js

```diff
@@ -0,0 +1,199 @@
+import GoogleAds from '../../../src/integrations/GoogleAds/browser';
+import { shouldSendEvent, getConversionData } from '../../../src/integrations/GoogleAds/utils';
+import {
+  mockEvents,
+  mockOrderId,
+  productAdded,
+  orderCompleted,
+  googleAdsTrack,
+  googleAdsConfigs,
+  trackCallPayload,
+  mockConversionId,
+  mockEventTypeConversions,
+} from './__mocks__/data';
+
+beforeAll(() => {});
+
+afterAll(() => {
+  jest.restoreAllMocks();
+});
+
+describe('GoogleAds init tests', () => {
+  let googleAds;
+
+  test('Testing init call of Google Ads with ConversionId', () => {
+    googleAds = new GoogleAds({ conversionID: 'AW-11071053757' }, {});
+    googleAds.init();
+    expect(typeof window.gtag).toBe('function');
+  });
+});
+
+describe(googleAdsTrack, () => {
+  let googleAds;
+  beforeEach(() => {
+    googleAds = new GoogleAds(googleAdsConfigs[0], {});
```

Comment:
Shouldn't we be passing mocked analytics object as second argument ?
ref : https://github.com/rudderlabs/rudder-sdk-js/blob/8f903afcd3a6d5f277fa63c9c24929ce93602456/__tests__/integrations/FacebookPixel/FacebookPixel.test.js#L22

Reply:
basically we are passing destination config as first argument and we don't need mockAnalytics here so we are passing empty object

Reply:
What about the scenarios where we are testing actual `track` events

Reply:
this test cases are for actual track events only